<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Brush in a Rush</string>
  </property>
  <property name="windowIcon">
   <iconset resource="backGround.qrc">
    <normaloff>:/icons/painting-1.png</normaloff>
    <normalon>:/icons/painting-1.png</normalon>:/icons/painting-1.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 100);
color: rgb(255, 255, 255);
selection-background-color: rgb(167, 167, 167);</string>
   </property>
   <widget class="QMenu" name="menuedrt">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSearch"/>
    <addaction name="actionSort"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Draw</string>
    </property>
    <widget class="QMenu" name="menuDraw">
     <property name="title">
      <string>Shapes...</string>
     </property>
     <addaction name="actionCircle"/>
     <addaction name="actionRectangle"/>
     <addaction name="actionStraightLine"/>
    </widget>
    <addaction name="menuDraw"/>
    <addaction name="separator"/>
    <addaction name="actionPen_color"/>
    <addaction name="actionFill_color"/>
    <addaction name="actionPen_size"/>
    <addaction name="separator"/>
    <addaction name="actionFill_with_color"/>
    <addaction name="actionEraser"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuClicl_me">
    <property name="title">
     <string>Click me</string>
    </property>
    <addaction name="actionClick"/>
   </widget>
   <addaction name="menuedrt"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdd"/>
   <addaction name="menuClicl_me"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStraightLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionPen_color"/>
   <addaction name="actionPen_size"/>
   <addaction name="actionFill_color"/>
   <addaction name="separator"/>
   <addaction name="actionEraser"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClear"/>
   <addaction name="actionSearch"/>
   <addaction name="actionSort"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/open-folder.png</normaloff>:/icons/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/add-file.png</normaloff>:/icons/add-file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPen_color">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/colored-pencils.png</normaloff>:/icons/colored-pencils.png</iconset>
   </property>
   <property name="text">
    <string>Pen color</string>
   </property>
  </action>
  <action name="actionPen_size">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/pencil-and-ruler.png</normaloff>:/icons/pencil-and-ruler.png</iconset>
   </property>
   <property name="text">
    <string>Pen size</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionFill_with_color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill with color</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/eraser.png</normaloff>:/icons/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/icons8-filled-circle-100.png</normaloff>:/icons/icons8-filled-circle-100.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/icons8-rectangle-100.png</normaloff>:/icons/icons8-rectangle-100.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFill_color">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/paint-bucket.png</normaloff>:/icons/paint-bucket.png</iconset>
   </property>
   <property name="text">
    <string>Fill color</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionStraightLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/icons8-line-100.png</normaloff>:/icons/icons8-line-100.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Shape</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/cleaning.png</normaloff>:/icons/cleaning.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionClick">
   <property name="text">
    <string>Click</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="icon">
    <iconset resource="backGround.qrc">
     <normaloff>:/icons/sort.png</normaloff>:/icons/sort.png</iconset>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="backGround.qrc"/>
 </resources>
 <connections/>
</ui>
